data/human/news.txt

```diff
@@ -801,6 +872,25 @@ news "geological engineer"
 
 
 
+news "electrical engineer"
+	location
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	name
+		word
+			"Electrical engineer"
+			"Power networks expert"
+	message
+		word
+			`"`
+		word
+			"Earth has a weird power-distribution network, where the polarity of home outlets changes 50 or 60 times per second. Every other planet just uses a positive and negative polarity on outlets."
```

Comment:
```suggestion
			"Earth has a weird power-distribution network, where the polarity of home outlets changes 50 or 60 times per second. Every other planet just uses a positive and negative polarity in outlets."
```
It's unclear what the exact preposition would be here, but I think "on" has other implications (do you rest the plug on something and it's "plugged in"?)

Reply:
`For` sounds good. I added both versions.